pom.xml

```diff
@@ -184,6 +183,17 @@
         <scope>test</scope>
       </dependency>
     </dependencies>
+
+    <dependencyManagement>
```

Comment:
I am not sure what is the intention here. This artifact is not really consumed anywhere

Reply:
I had to add this because of some dependency upgrade which was breaking the enforcer-rule.

Reply:
This looks to avoid:

```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for commons-net:commons-net:3.5 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.main:maven-plugin:3.1.2
    +-org.apache.maven.wagon:wagon-ftp:3.0.0
      +-commons-net:commons-net:3.5
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-test-harness:2.49
    +-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.31-2
      +-commons-net:commons-net:3.6
```

and the even latest `maven-plugin` depends on `commons-net-3.5`.

Though I don’t know the best practices for avoiding `RequireUpperBoundDeps`, it seems preferable to use `exclusion` rather than `dependencyManagement` as it should work even when future jenkins-test-harness depends on `commons-net-3.7`.